contrib/terraform/openstack/README.md

```diff
@@ -21,9 +21,7 @@ most modern installs of OpenStack that support the basic services.
 - [Ultimum](https://ultimum.io/)
 - [VexxHost](https://vexxhost.com/)
 - [Zetta](https://www.zetta.io/)
-
-### Known incompatible public clouds
-- T-Systems / Open Telekom Cloud: requires `wait_until_associated`
+- [Open Telekom Cloud](https://cloud.telekom.de/) : requires to set the variable `wait_for_floatingip = "true"` in your cluster.tf
```

Comment:
Wouldn't this PR solve deploying on Open Telekom Cloud? I think you can move this row up to compatible clouds then, as long as the notice is there.

Reply:
I think so too. And another rebase since there is a merge conflict :(

Reply:
I put the Open Telekom Cloud on the compatible cloud list.